edd-blockonomics.php

```diff
@@ -673,14 +680,16 @@ function show_basic() {
           '25' => '25',
           '30' => '30'
         ),
-        'class' => 'edd-blockonomics-advanced'
+        'class' => 'edd-blockonomics-advanced',
+        'default' => '10'
       ),
       array(
         'id'      => 'edd_blockonomics_margin',
         'name'    => __('Extra Currency Rate Margin % (Increase live fiat to BTC rate by small percent)', 'edd-blockonomics'),
         'type'    => 'number',
         'max'     => 20,
-        'class' => 'edd-blockonomics-advanced'
+        'class' => 'edd-blockonomics-advanced',
+        'default' => 0
```

Comment:
Using default value as `0` does not work since the edd_update_option function checks if value is `empty`: https://github.com/awesomemotive/easy-digital-downloads/blob/main/includes/admin/settings/register-settings.php#L65-L69
We can rather use `'default' => 'zero'` and apply our own filter to change this value to 0:
```
function edd_blockonomics_update_option_filter($value, $key) {
  if ($key === 'edd_blockonomics_underpayment_slack' || $key === 'edd_blockonomics_margin') {
      if ($value === 'zero') {
          $value = 0;
      }
  }

  return $value;
}

add_filter('edd_update_option', 'edd_blockonomics_update_option_filter', 10, 2);
```